### DISCUSSION TITLE
Integration with mindsdb

### DISCUSSION THREAD

**Discussion by MichaelScofield** [GreptimeTeam/greptimedb#3563] [Ideas]
### What problem does the new feature solve?

More integrations.

### What does the feature do?

Integration with [mindsdb](https://github.com/mindsdb/mindsdb)

**Comment by tisonkun** [comment]
Moved to Ideas until we have more concrete proposals.

Would you please elaborate more about what directions of the integration? How GreptimeDB uses this software or how GreptimeDB is used by it.

**Comment by killme2008** [comment]
Looks like we can use the MySQL handler, and @sunng87 Already created a document for it 

https://github.com/GreptimeTeam/docs/pull/895
